app/scripts/services/Municipalities.svc.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+angular.module('yourStlCourts').factory('Municipalities', function ($resource, $q) {
+  var allMuniResource = $resource('municipalities');
+  var MuniResource = $resource('municipalities/:id');
```

Comment:
This is fine, but for future reference, you don't need both of these. Just the second one will work fine for both